docs/semgrep-supply-chain/license-scanning.md

```diff
@@ -0,0 +1,115 @@
+---
+slug: license-scanning
+append_help_link: true
+title: License scanning 
+hide_title: true
+description: "Semgrep Supply Chain can detect and list a package's license. Prevent or exempt certain packages from being used based on its license."
+tags:
+  - Team & Enterprise tier
+  - Semgrep Supply Chain
+---
+
+# License scanning
+
+Semgrep Supply Chain's **license scanning** feature enables you to explicitly allow or disallow (block) a package's use in your repository based on its license. For example, your company policy may disallow the use of packages with the Creative Commons Attribution-NonCommercial (CC-BY-NC) license.
+
+![Screenshot of Semgrep Supply Chain Dependencies tab with licenses listed](/img/sc-license-scanning.png)
+*Figure 1*. Screenshot of Semgrep Supply Chain Dependencies tab with licenses listed.
+
+:::info Prerequisites
+* License scanning can accessed through Semgrep Cloud Platform (SCP). [Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to view and manage license workflows.
+* To detect licenses, you must [add or onboard a project](/semgrep-code/getting-started/#option-b-adding-a-repository-from-github-gitlab-or-bitbucket) (repository) to Semgrep Cloud Platform for scanning.
```

Comment:
I would add one more note saying that the user **must** have Dependency Search enabled (we need to store/index lockfile content to show relevant licenses)

Reply:
Thanks for pointing this out. This simplifies the steps (basically just log in and click dependencies); I've updated dependency search instead with the instructions on where to find the toggle.